tracer/build/_build/docker/centos7.build.dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM centos:7 as base
 
 RUN yum update -y \
+    && yum install -y centos-release-scl \
```

Comment:
😭 Why do you hate me? Now I need to redo all the VM images to fix the caching :try_not_to_cry: 😛

Reply:
So sorry, and I cannot assure that this will be the last time 😬